src/frontend/views/contributions_view.py

```diff
@@ -0,0 +1,24 @@
+from django.views.generic import ListView
+from django.shortcuts import get_object_or_404
+from django.http import Http404
+
+from src.apps.runs.models import Run
+from src.apps.users.models import User
+
+from . import view_utils
+
+class ContributionsView(ListView):
+  template_name = "pages/contributions.html"
+  context_object_name = "user_list"
+
+  def get_queryset(self):
+    view_utils.set_current_run_or_run_from_url_for_view_get_queryset(self)
```

Comment:
why isn't the run explicit ?

Idealy we would have diiferents views:
- one for `fontend/run/<run>/contributions` where run is the run to display and may be the current run (by default, links redirect to `run/test/networks` when `test` is the active run
- one for `fontend/contributions` which don't filter by active run and display the contribution for all runs (no filter)

Reply:
I thought about and agree, I went with just making the contributors page look at all games from all runs.